waku/node/config.nim

```diff
@@ -0,0 +1,170 @@
+import
```

Comment:
Is this a bridge specific config? If so we might want to make it explicit (e.g. bridge folder or bridge in name). If it is a general config, how does this relate to v1/config and v2/config?

Reply:
Yes it is specific for the bridge application, so it's similar as options for v1 and v2 but I only kept the options that make sense for a bridge (e.g. no light node settings for v1). Thought about putting it all in a subfolder but there are already so many folders :). I think I'll rename it.